src/pages/settings/InitialSettingsPage.js

```diff
@@ -252,14 +238,14 @@ class InitialSettingsPage extends React.Component {
                 translationKey: 'initialSettingsPage.signOut',
                 icon: Expensicons.Exit,
                 action: () => {
-                    this.signout(false);
+                    signout(false);
                 },
             },
         ];
-    }
+    };
 
-    getMenuItem(item, index) {
-        const keyTitle = item.translationKey ? this.props.translate(item.translationKey) : item.title;
+    const getMenuItem = (item, index) => {
```

Comment:
and this one too.  Basically when we have methods that will affect the render, are dependent on the props and not on internal state, we can wrap them with `useMemo`.